tests/datastore_emulator/token_store_test.py

```diff
@@ -1,43 +1,29 @@
 import unittest
 
-# Imports might be highlighted as "unused" by IntelliJ, but they are used, see setUp()
-from google.appengine.api import datastore
-from google.appengine.api import memcache
-from google.appengine.ext import testbed
-from google.appengine.ext import ndb
-from token_store import TokenStore
-from refresh_token import RefreshToken
+from bond_app.token_store import TokenStore, RefreshToken
 from datetime import datetime
+from tests.datastore_emulator import datastore_emulator_utils
 
 provider_name = "test"
 
 
 class TokenStoreTestCase(unittest.TestCase):
 
     def setUp(self):
-        # First, create an instance of the Testbed class.
-        self.testbed = testbed.Testbed()
-        # Then activate the testbed, which prepares the service stubs for use.
-        self.testbed.activate()
-        # Next, declare which service stubs you want to use.
-        self.testbed.init_datastore_v3_stub()
-        self.testbed.init_memcache_stub()
+        # Make sure to run these tests with a Datastore Emulator running or else they will fail with 'InternalError.'
+        # See the README in this directory.
+        datastore_emulator_utils.setUp(self)
 
         self.user_id = "abc123"
         self.token_str = "aaaaaabbbbbbcccccccddddddd"
         self.issued_at = datetime.now()
```

Comment:
Do want t fixed fake time?

Reply:
I think this is fine how it is. The tests just check that the token store is saving and loading the same value for `issued_at`, so this will just be a fixed value when the tests are run.